### PR TITLE
Branch edit uml

### LINKED ISSUE #98: high level UML for Model
Create uml diagram for high level description for models in DG

### PR DESCRIPTION
**Pull request by QinHaichen12**
Please review pls pls pls beg

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Features
 
-<box type="info" seamless>
+<box type="info" seamless></box>
```
Comment (keithxun) [inline docs/UserGuide.md]: Is this the correct place to close the box

File: docs/diagrams/ModelClassDiagram.puml [hunk 2]
```diff
@@ -40,6 +42,9 @@ Member *--> Name
 Member *--> Telegram
 Member *--> Address
 Member *--> "*" Tag
+Member *--> "*" Session
+
```
Comment (keithxun) [inline docs/diagrams/ModelClassDiagram.puml:46]: Weird empty line

File: docs/team/qinhaichen12.md [hunk 3]
```diff
@@ -12,32 +9,30 @@
...
+  - Decent scorer for Data Structures and Algorithms and Programming Methodology II
+  - Obtained top 8 (only 16 teams) in a robotics contest organized by the School of Computing
 - Bachelor of Science in Mathematics
-  - Current GPA: 5.00/5.00 (Highest Distinction)
-  - Obtained highest grade (A+) in Calculus, Linear Algebra, and Probability
+  - Current GPA: not enough to be proud of
+  - Maybe rejected from SPM application (I love u Chin Chee Whye)
 
-### NUS High School of Mathematics and Science
+### NOT RAFFLES INSTITUTION
```
Comment (keithxun) [inline docs/team/qinhaichen12.md:19]: I like how you did this

File: docs/team/qinhaichen12.md [hunk 4]
```diff
@@ -12,32 +9,30 @@
...
-- Collaborated with a team of four to create a mobile app with Unity engine and C# incorporating AR functionalities within 3 days and trained a Machine Learning model using YOLO_v8 for animal image recognition leveraging Python and TensorFlow.
-- Deployed a full-stack web application on Vercel platform that enables users to create and participate in prediction challenges using Next.js for frontend and Supabase for backend database management.
+- Figured out that $2.50 economic rice exists in Terrace
+- Convinced my friends that deck is the best canteen
 
 ## Technical Skills
 
-- **Languages**: Java, Python, Javascript, PostgreSQL, C#, C++
-- **Frameworks**: React, Next.js
+- Yap
```
Comment (taggyhan) [inline docs/team/qinhaichen12.md:33]: Usage of Specific Non-oxford dictionary terms should be avoided

File: docs/team/qinhaichen12.md (hunk 3 above)
Comment (taggyhan) [reply docs/team/qinhaichen12.md:19]: Is "not-ri" something to include?

**Pull request merged**